index.js

```diff
@@ -59,6 +59,23 @@ module.exports = {
     // Prefer named export
     'import/prefer-default-export': 'off',
     'import/no-default-export': 'error',
+
+    // Order import with line separation between groups
+    'import/order': [
+      'error',
```

Comment:
`warning` for now, this is not obligation for me.
See to set `error` in another major version.